src/logging.cc

```diff
@@ -628,7 +628,7 @@ inline void LogDestination::RemoveLogSink(LogSink *destination) {
   MutexLock l(&sink_mutex_);
   // This doesn't keep the sinks in order, but who cares?
   if (sinks_) {
-    for (int i = sinks_->size() - 1; i >= 0; i--) {
+    for (int i = (int)sinks_->size() - 1; i >= 0; i--) {
```

Comment:
`ssize_t` or `ptrdiff_t` instead of the type cast to `int` makes more sense to avoid overflows.